src/pages/groupComparison/GroupComparisonPage.tsx

```diff
@@ -312,7 +314,7 @@ export default class GroupComparisonPage extends React.Component<
                             <MethylationEnrichments store={this.store} />
                         </MSKTab>
                     )}
-                    {this.store.showGenericAssayTab &&
+                    {/* {this.store.showGenericAssayTab &&
```

Comment:
We can remove unused code when you think you won't enable it anymore.

Reply:
Okay, done